docs-src/cloud-context/pricing.md

```diff
@@ -0,0 +1,100 @@
+---
+id: pricing
+title: Temporal Cloud Pricing
+sidebar_label: Pricing
+description: "Temporal Cloud pricing information."
+tags:
+  - guide-context
+---
+
+Temporal Cloud is a consumption-based service; you pay only for what you need when you need it. Pricing is flexible, transparent, and predictable, so you know your costs and never pay for unused components.
+
+This page introduces the elements of Temporal Cloud pricing so that you can estimate costs for your implementation. To gain a reliable estimate, [contact our team](https://pages.temporal.io/contact-us).
+
+## Temporal Actions
+
+Actions are the fundamental consumption pricing unit in Temporal Cloud. An Action in Temporal occurs as part of an execution of your Workflow. Each time you execute a Temporal Workflow (a Workflow Execution), the associated Actions are collected and ultimately represent the state and progress of your Temporal Application.
+
+For a deeper description of Actions, please refer to [this section](/cloud#action) of our documentation. You can also [reach out to our team](https://pages.temporal.io/contact-us) to get more information or to help size your number of Actions.
```

Comment:
```suggestion
For a deeper description of Actions, see [Action](/cloud#action) of our documentation. You can also [reach out to our team](https://pages.temporal.io/contact-us) to get more information or to help size your number of Actions.
```
- https://developers.google.com/style/word-list#please
- https://developers.google.com/style/link-text#write-link-text
- Referring to "our documentation" when using the documentation.